### ISSUE TITLE
Error while launching the language server

### ISSUE THREAD

**Issue by jcs090218** [znck/grammarly#234]
I have followed the Development Setup,

```
pnpm install
pnpm run build
```

But I have encountered several errors:

```
extension\src\index.ts → extension/dist/extension/index.browser.js...
> node_modules/.pnpm/minimatch@3.1.2/node_modules/minimatch/minimatch.js:4:47: error: Plugin "rollup" returned a non-absolute path: path (set a namespace if this is not a file path)
4 │ var path = (function () { try { return require('path') } catch (e) {}}()) || {
╵                                                ~~~~~~

[!] (plugin esbuild) Error: Build failed with 1 error:
node_modules/.pnpm/minimatch@3.1.2/node_modules/minimatch/minimatch.js:4:47: error: Plugin "rollup" returned a non-absolute path: path (set a namespace if this is not a file path)
Error: Build failed with 1 error:
node_modules/.pnpm/minimatch@3.1.2/node_modules/minimatch/minimatch.js:4:47: error: Plugin "rollup" returned a non-absolute path: path (set a namespace if this is not a file path)
at failureErrorWithLog (C:\Users\xxx\Downloads\workspace\grammarly\node_modules\.pnpm\esbuild@0.12.29\node_modules\esbuild\lib\main.js:1475:15)
at C:\Users\xxx\Downloads\workspace\grammarly\node_modules\.pnpm\esbuild@0.12.29\node_modules\esbuild\lib\main.js:1133:28
at runOnEndCallbacks (C:\Users\xxx\Downloads\workspace\grammarly\node_modules\.pnpm\esbuild@0.12.29\node_modules\esbuild\lib\main.js:923:63)
at buildResponseToResult (C:\Users\xxx\Downloads\workspace\grammarly\node_modules\.pnpm\esbuild@0.12.29\node_modules\esbuild\lib\main.js:1131:7)
at C:\Users\xxx\Downloads\workspace\grammarly\node_modules\.pnpm\esbuild@0.12.29\node_modules\esbuild\lib\main.js:1240:14
at C:\Users\xxx\Downloads\workspace\grammarly\node_modules\.pnpm\esbuild@0.12.29\node_modules\esbuild\lib\main.js:611:9
at handleIncomingPacket (C:\Users\xxx\Downloads\workspace\grammarly\node_modules\.pnpm\esbuild@0.12.29\node_modules\esbuild\lib\main.js:708:9)
at Socket.readFromStdout (C:\Users\xxx\Downloads\workspace\grammarly\node_modules\.pnpm\esbuild@0.12.29\node_modules\esbuild\lib\main.js:578:7)
at Socket.emit (node:events:526:28)
at addChunk (node:internal/streams/readable:315:12)
```

And if I tried to start the language server, I get:

```
C:\Users\xxx\Downloads\workspace\grammarly\packages\grammarly-languageserver\dist\index.cjs.js:6
const fetch = require('node-fetch');
^

Error [ERR_REQUIRE_ESM]: require() of ES Module C:\Users\xxx\Downloads\workspace\grammarly\node_modules\.pnpm\node-fetch@3.2.4\node_modules\node-fetch\src\index.js from C:\Users\xxx\Downloads\workspace\_backup\grammarly\packages\grammarly-languageserver\dist\index.cjs.js not supported.
Instead change the require of index.js in C:\Users\xxx\Downloads\workspace\grammarly\packages\grammarly-languageserver\dist\index.cjs.js to a dynamic import() which is available in all CommonJS modules.
at Object.<anonymous> (C:\Users\xxx\Downloads\workspace\grammarly\packages\grammarly-languageserver\dist\index.cjs.js:6:15)
at Object.<anonymous> (C:\Users\xxx\Downloads\workspace\grammarly\packages\grammarly-languageserver\bin\server.js:3:16) {
code: ←[32m'ERR_REQUIRE_ESM'←[39m
```

Any idea are appreciated!

**Comment by znck** [comment]
In rollup.config.js, I am using unix paths.

**Comment by jcs090218** [comment]
Thanks! ❤️ The first issue is resolved after merging upstream.

But still could not get rid of the second error while starting the language server! 😖

**Comment by jcs090218** [comment]
The next error is:

```
node:internal/process/promises:279
triggerUncaughtException(err, true /* fromPromise */);
^

Error [ERR_REQUIRE_ESM]: require() of ES Module C:\Users\xxx\Downloads\workspace\grammarly-language-server\node_modules\.pnpm\node-fetch@3.2.4\node_modules\node-fetch\src\index.js from C:\Users\xxx\Downloads\workspace\grammarly-language-server\packages\grammarly-languageserver\dist\index.cjs.js not supported.
Instead change the require of index.js in C:\Users\xxx\Downloads\workspace\grammarly-language-server\packages\grammarly-languageserver\dist\index.cjs.js to a dynamic import() which is available in all CommonJS modules.
at C:\Users\xxx\Downloads\workspace\grammarly-language-server\packages\grammarly-languageserver\dist\index.cjs.js:37:145
at async C:\Users\xxx\Downloads\workspace\grammarly-language-server\packages\grammarly-languageserver\dist\index.cjs.js:37:64 {
code: 'ERR_REQUIRE_ESM'
}
```

Let me know if I should stop informing you the new error! I am not trying burden you on this error! 😖 But I think it would be nice to have someone to help you with the test? 😅

**Comment by znck** [comment]
Continue reporting 😅

I guess I need to separate web extension.

**Comment by jcs090218** [comment]
I think the error is caused by `node-fetch` v3. Any chance to use the previous `node-fetch` version `^2.6.1`? 😖

**Comment by znck** [comment]
Yes. It will be fixed by #240
